suite-native/biometrics/src/biometricsAtoms.ts

```diff
@@ -7,6 +7,11 @@ const isBiometricsOptionEnabledAtom = atomWithUnecryptedStorage<boolean>(
     false,
 );
 
+const isBiometricsInitialSetupFinishedAtom = atomWithUnecryptedStorage<boolean>(
+    'hasDecidedBiometrics',
```

Comment:
Please rename this to reflect name of atom

Reply:
renamed in fc97459